contracts/utils/RangePoolManager.sol

```diff
@@ -127,6 +128,7 @@ contract RangePoolManager is
         if (feeTiers[swapFee] != 0) {
             revert FeeTierAlreadyEnabled();
         }
+        if(tickSpacing <= 0 || tickSpacing >= 16384) revert FeeTierTickSpacingInvalid();
         feeTiers[swapFee] = tickSpacing;
```

Comment:
The upper end check of this is quite crazy.

A tick spacing of 16384 implies a 500% move.